.github/CONTRIBUTING.md

```
@@ -0,0 +1,55 @@
+# Contributing to KyoriAdventureDSL
+
+Thanks for your interest in contributing! This project is in **pre-alpha**, so the surface area changes often — small, focused contributions are the most useful.
+
+## Be respectful
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)